package.json

```diff
@@ -25,8 +25,9 @@
     "mustache": "^2.1.3",
     "node-bourbon": "^4.2.3",
     "node-neat": "^1.7.2",
+    "node-normalize-scss": "^1.0.3",
     "nodemon": "^1.4.1",
-    "normalize.css": "^3.0.3"
+    "normalize.scss": "^0.1.0"
```

Comment:
The change removes one of the libs and replaces with the sass version, and it's used [here](https://github.com/mozilla/idea-town/pull/119/files#diff-1fbf9d34e7bbb1cc8e088ac2d7105c4eR1)